### DISCUSSION TITLE
Multivariate Time Series Regression

### DISCUSSION THREAD

**Discussion by nils-fl** [alan-turing-institute/sktime#1828] [Q&A]
Hi,

I was wondering if there is a simple way to go from multivariate classification to multivariate regression by only changing the labels from classes to continuous values? If so, would the `ComposableTimeSeriesForestRegressor` be the right model? Could you give me a hint on how to use it?

Cheers
Nils

**Comment by BradKML** [comment]
Thinking about this https://www.sktime.org/en/stable/examples/01_forecasting.html?highlight=multivariate#1.2.4.-Multivariate-forecasting

**Comment by fkiraly** [comment]
> I was wondering if there is a simple way to go from multivariate classification to multivariate regression by only changing the labels from classes to continuous values?

Not sure what you mean - could you add some details or math?

It seems easier to go the other way round, i.e., turn a regression problem to classification by binning or similar.

**Reply by BradKML** [reply]
Maybe the categories are ordinal (e.g. education attainment, rank of a position)?
